tests/manager/manager-set-uri-options-002.phpt

```diff
@@ -47,5 +47,6 @@ printf("Inserted: %d\n", $inserted);
 <?php exit(0); ?>
 --EXPECTF--
 OK: Got MongoDB\Driver\Exception\ConnectionTimeoutException
+No suitable servers found (`serverSelectionTryOnce` set): [connection closed calling ismaster on '%s']
```

Comment:
Ah, I didn't realize and/or forgot that server selection failures are ConnectionTimeoutExceptions. That's clear from the logic in `phongo_exception_from_mongoc_domain()`, though.

I will take this moment to update the docs on http://php.net/manual/en/class.mongodb-driver-exception-connectiontimeoutexception.php.

Reply:
Updated: http://svn.php.net/viewvc?view=revision&revision=344498